containers/python-examples/NLTK/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.4"
+
+services:
+  grpc-api:
+    image: ghcr.io/deephaven/nltk-base:${VERSION:-latest}
```

Comment:
This is not the correct image. It should be `server-nltk`. See packages available at https://github.com/orgs/deephaven/packages?repo_name=deephaven-core, this comment also applies to all the other compose files. Note: it looks like we need to tag the extended images with latest, filed an issue for myself https://github.com/deephaven/deephaven-core/issues/1808

Reply:
All naming has been updated.  Thanks for pointing that out